### PR TITLE
fix import consolidation

### PR DESCRIPTION
**Pull request by BitBaseBit**

### CODE REVIEW

File: libwarp/solidity_prepass/WarpVisitor.cpp [hunk 1]
```diff
@@ -311,6 +314,9 @@ void WarpVisitor::setCompilerOptions() {
 OptimiserSettings WarpVisitor::optimizerSettings() {
     string yulOptimiserSteps = OptimiserSettings::DefaultYulOptimiserSteps;
     erase(yulOptimiserSteps, 'i');  // remove FullInliner
+    erase(yulOptimiserSteps, 'v');  // remove FullInliner
+    erase(yulOptimiserSteps, 'F');  // remove FullInliner
+    yulOptimiserSteps += 'x';
```
Comment (swapnilraj) [inline libwarp/solidity_prepass/WarpVisitor.cpp:319]: Why are we adding expression splitting again?

File: libwarp/solidity_prepass/WarpVisitor.cpp [hunk 2]
```diff
@@ -87,6 +87,9 @@ void WarpVisitor::writeModifiedSolidity() {
 CommandLineInterface WarpVisitor::getCli(char const* sol_filepath) {
     string yulOptimiserSteps = OptimiserSettings::DefaultYulOptimiserSteps;
     erase(yulOptimiserSteps, 'i');  // remove FullInliner
+    erase(yulOptimiserSteps, 'v');  // remove FullInliner
+    erase(yulOptimiserSteps, 'F');  // remove FullInliner
+    yulOptimiserSteps += 'x';
```
Comment (murcake) [inline libwarp/solidity_prepass/WarpVisitor.cpp]: These are not used anywhere, perhaps, it's better delete them altogether, they are misleading

File: libwarp/solidity_prepass/WarpVisitor.cpp (hunk 1 above)
Comment (murcake) [reply libwarp/solidity_prepass/WarpVisitor.cpp]: The comments are also not correct

**Review by murcake — COMMENTED** [review]
What does this PR do? Can you please show what it fixes?

**Comment by BitBaseBit** [comment]
> What does this PR do? Can you please show what it fixes?

When we remove expression splitting, it broke some of the functionality of YulVisitor, where we were constructing `fun_warp_constructor_DynArgs` incorrectly.

**Review by murcake — APPROVED** [review]

**Pull request merged**